### DISCUSSION TITLE
(highly) Customizing the UI

### DISCUSSION THREAD

**Discussion by wylited** [flxzt/rnote#676] [Q&A] [answered]
I wanted to customize the ui (ie. remove certain rounded corners, change colors, replace items, e.t.c.)
How can I get started with this?
Is there a css file I can customize before building that follows gtk css or something simmilar?
Or is there a method for the config where it injects the css on load?

thanks in advance.

**Comment by flxzt** [comment, marked as answer]
If you are looking to tweak the UI via css: there is the [style.css](https://github.com/flxzt/rnote/blob/bc73bb352d7070b2e398c533c0748a03b9dc2c93/rnote-ui/data/ui/style.css) app stylesheet, which gets loaded in [here](https://github.com/flxzt/rnote/blob/bc73bb352d7070b2e398c533c0748a03b9dc2c93/rnote-ui/src/appwindow/imp.rs#L123). Depending on what you want to do, you also have to modify the `.ui` files to add css classes to widgets, adjust margins, change layout, ... . They are located [here](https://github.com/flxzt/rnote/tree/bc73bb352d7070b2e398c533c0748a03b9dc2c93/rnote-ui/data/ui)
